espnet2/asr/decoder/transformer_decoder.py

```diff
@@ -128,6 +128,12 @@ def forward(
         memory_mask = (~make_pad_mask(hlens, maxlen=memory.size(1)))[:, None, :].to(
             memory.device
         )
+        # Padding for Longformer
```

Comment:
@sw005320 - This is the change in the decoder, i.e., while creating the mask for the encoded features, make-pad-mask uses the longest input length, but the longest length may be more than that because of the attention padding in the encoder. 
So for any decoder- transformer ,transducer etc, this is the only change to use longformer self-attention in the encoder

Reply:
ok